glotaran/testing/simple_generator.py

```diff
@@ -0,0 +1,207 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from dataclasses import field
+from typing import TYPE_CHECKING
+from typing import Literal
+
+from glotaran.model import Model
+from glotaran.parameter.parameter_group import ParameterGroup
+
+if TYPE_CHECKING:
+    from glotaran.utils.ipython import MarkdownStr
+
+
+def _split_iterable_in_values_and_dicts(input_list: list) -> tuple[list, list]:
```

Comment:
The lists also need have a type e.g. `list[float]` and `list[dict[str, bool|float]]`.

Reply:
Resolved in ffc4f85cd951f26c51b0c1ae6c147bb61115f420